lua/nvim-tree/config.lua

```diff
@@ -67,9 +67,14 @@ function M.get_bindings()
     ["<C-v>"]          = M.nvim_tree_callback("vsplit"),
     ["<C-x>"]          = M.nvim_tree_callback("split"),
     ["<C-t>"]          = M.nvim_tree_callback("tabnew"),
+    ["<C-k>"]          = M.nvim_tree_callback("prev_sibling"),
+    ["<C-j>"]          = M.nvim_tree_callback("next_sibling"),
```

Comment:
hum i use those two for moving between up and down windows and i think many people would too. maybe something like `<` or `>` ?

Reply:
Great!